docs/docs/migrate-from-v1/README.md

```diff
@@ -0,0 +1,358 @@
+      headers: {
+        Authorization: `Bearer ${this.slack.$auth.oauth_access_token}`,
+      },
+    }
+  },
+})
+```
+
+After testing the step, you'll see the Slack app will appear in the **Configuration** section on the left hand side. In this section you can choose which Slack account you'd like to use in the step.
+
+<img src="./images/app-props-example.png" alt="Example of adding an app connection to a v2 Node.js step">
+
+
+### HTTP Response
+
+You can still return an HTTP response from an HTTP-triggered workflow.
+
+Use [`$.respond`](/docs/workflows/steps/triggers/#http) to send a JSON or string response from the HTTP call that triggered the workflow.
+
+```javascript
+export default defineComponent({
+  async run({ steps, $ }) {
+    $.respond({
+      status: 200,
+      headers: {},
+      body: { 
+        message: "hello world!"
+      }
+    });
+  },
+})
+```
+
+Please note, you'll also need to configure the HTTP trigger step to also allow custom responses. Use the dropdown in the **HTTP Response** section of the HTTP trigger to select the **Return a custom response from your workflow** option:
+
+<img src="./images/custom-http-response-option.png" alt="Select the option to allow your workflow to send it's own HTTP responses">
+
+## Known Gaps & Limitations
+
+However, some features from the original builder are not currently available in v2. The Pipedream team is working to quickly address these items, but if you have feedback that isn't listed here, please [reach out](/support).
+
+### Sharing workflows
+
+At this time, sharing is not yet implemented in v2 of the workflow builder. As workaround, create your workflows in a organization which make workflows available to your team members.
+
+If you need assistance transferring workflows across accounts, [please contact us](/docs/support).
+
+### `$checkpoint`
+
+The `$checkpoint` functionality to save data between workflow runs is not yet supported in v2.
+
+However, you can leverage the `$.service.db` service to store arbitrary data across your workflow runs like unique IDs. [Read more about using `$.service.db` here.](/code/nodejs/#managing-state)
+
+::: warning
+Please note that any values stored in `$.service.db` are only accessible in subsequent workflow runs _in the same step_.
+:::
+
+### Public workflows
+
+At this time, all v2 workflows are private. Unforunately at this time there is no workaround. We'll announce when a workaround for this limiation is available.
+
+If you're working with Pipedream support to troubleshoot your workflow, you can share it with the support team under your workflow's **Settings**.
+
+### Rolling back a specific version
+
+In v2, you can test and save your progress on a workflow _without_ deploying it.
+
+However, after deploying it's not possible to rollback to a prior version of a deployed workflow.
+
+You can still edit a deployed workflow, just like in v1 but automatic version rollbacks are not currently possible.
+
+## FAQs
```

Comment:
Re: the conversation standup, we should have an FAQ for something like "Where did my list of events on the left go?"

Reply:
Added this 👍